### PR TITLE
Get only trade events from last 1000 blocks

### PR DESCRIPTION
**Pull request by AugustoL**
- Adds the fromBlock value to be used finto the contracts.json file in src.
- Get only the trades form the last 1000 blocks.

### CODE REVIEW

File: scripts/loadDeployments.js [hunk 1]
```diff
@@ -23,15 +23,15 @@ async function loadDeployment(network) {
   let contractsDeployed = {'contracts': {}};
   if (fs.existsSync('src/config/contracts.json'))
     contractsDeployed = JSON.parse(fs.readFileSync('src/config/contracts.json', 'utf-8'));
-
   const datContract = DATContract.at(proxies['BC-DAPP/DecentralizedAutonomousTrust'][0].address);
-
+  const fromBlock = (network == 'mainnet') ? 10000000 : (network == 'kovan') ? 18000000 : 0;
```
Comment (pimato) [inline scripts/loadDeployments.js:27]: why starting fromBlock from 10000000 and not 
10012634 which was the block that deployed the contract?

https://etherscan.io/tx/0x5ba97e7ceb6b409267e8a07d33dbc0a814ef5ea3eae9e2077ecea2b9ccc60a6d

File: src/stores/datStore.ts [hunk 2]
```diff
@@ -404,8 +406,13 @@ export default class DatStore {
         datAddress: string,
         numToGet: number
     ): Promise<TradeEvent[]> {
-        const buyEvents = await this.fetchBuyEvents(datAddress, numToGet);
-        const sellEvents = await this.fetchSellEvents(datAddress, numToGet);
+        const { library } = this.rootStore.providerStore.getActiveWeb3React()
```
Comment (pimato) [inline src/stores/datStore.ts:409]: missing `;`

**Comment by AugustoL** [comment]
@pimato changes requested done

File: src/stores/datStore.ts [hunk 3]
```diff
@@ -84,6 +84,8 @@ export interface SellReturn {
 
 export type TradeEvent = BuyEvent | SellEvent;
 
+export const TRADES_FROM_LAST_BLOCKS = 1000;
```
Comment (jpkcambridge) [inline src/stores/datStore.ts:87]: Should this be named `BLOCKS_TO_GET_TRADES_FROM` or `BLOCK_FETCH_NUMBER` or maybe `FROM_BLOCK_OFFSET`

**Pull request merged**